components/library/libxml2/Makefile

```diff
@@ -63,18 +64,18 @@ CONFIGURE_OPTIONS += --sysconfdir=$(ETCDIR)
 CONFIGURE_OPTIONS += --localstatedir="$(VARDIR)"
 CONFIGURE_OPTIONS += --disable-static
 CONFIGURE_OPTIONS += --with-pic
-CONFIGURE_OPTIONS += --with-icu
+# dont't re-enable this option again without good reasons,
+# it looks like it is breaking the packaging system: pkg
+# CONFIGURE_OPTIONS += --with-icu
 CONFIGURE_OPTIONS += --with-ftp
```

Comment:
Please change the warning. The dependency doesn't break pkg per se but only during upgrades of icu.
Furthermore, I would expect that the dependency will vanish. In your Makefile and pkg5 file the dependency is still there.